README.md

```diff
@@ -1,24 +1,192 @@
-# README
+<!-- TABLE OF CONTENTS -->
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# 📗 Table of Contents
 
-Things you may want to cover:
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [🚀 Live Demo](#live-demo)
+    - [Live video](#live-video)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Author](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
 
-* Ruby version
+<!-- PROJECT DESCRIPTION -->
 
-* System dependencies
+# 📖 EXPENSES TRACKER APP <a name="about-project"></a>
 
-* Configuration
+> Expenses-tracker-app is an application that allows the user to:
 
-* Database creation
+> - register and log in, so that the data is private to them.
+> - introduce new transactions associated with a category.
+> - see the money spent on each category.
 
-* Database initialization
+It is basically a mobile web application where you can manage your budget:
+you have a list of transactions associated with a certain category,
+so that you can see how much money you spent and on what.
 
-* How to run the test suite
+**
+expenses-tracker-app** is a Ruby on Rails application...
 
-* Services (job queues, cache servers, search engines, etc.)
+## 🛠 Built With <a name="built-with"></a>
 
-* Deployment instructions
+### Tech Stack <a name="tech-stack"></a>
 
-* ...
+<details>
+  <summary>Client / Server</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+    <li><a href="https://getbootstrap.com/">Bootstrap</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Deploy</summary>
+  <ul>
+    <li><a href="https://render.com/">Render</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database </summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Tests</summary>
+  <ul>
+    <li><a href="https://rspec.info/">Rspec</a></li>
+    <li><a href="https://github.com/teamcapybara/capybara">Capibara</a></li>
+  </ul>
+</details>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link]()
+
```

Comment:
- [ ] [REQUIRED] Good job so far! I think you need to deploy your project, as the [previous code reviewer](https://github.com/basitali111/Budget-App/pull/1#discussion_r1045108615) mentioned. Please follow the requirements mentioned by the [previous code reviewer](https://github.com/basitali111/Budget-App/pull/1#discussion_r1045108615). You should deploy the application using `Heroku`, `Netlify`, or `GitHub Pages`. The project should be deployed properly, and the `home` and `detail` page works as expected. A link to the deployed project should be present in your `README` 🙏🏼

Reply:
Please read it my code review did't even seen it its last hour and i am not in stage of implemented deployment from netify and github pages because its not react or javascript 
Hello dear code reviewer, just wanted to let you know that I have implemented all the requested changes from the previous reviewer except deploying the website on heroku, since Heroku is no longer free and my credit card was not accepted there, I cannot deploy it somewhere else due the deadlines, I have also reached out to student success, please kindly take into consideration this comment, happy code reviewing!
cheers 🥂